crates/env/src/api.rs

```diff
@@ -175,15 +175,19 @@ where
 
 /// Returns information about the required deposit and resulting rent.
 ///
+/// # Parameters
+///
+/// - `at_refcount`: The refcount assumed for the returned `custom_refcount_*` fields.
```

Comment:
Document what is the semantic of it being `None`?
Also briefly document what is the effect of this parameter.